common/xrdp_client_info.h

```diff
@@ -228,6 +243,9 @@ struct xrdp_client_info
 
     // Can we resize the desktop by using a Deactivation-Reactivation Sequence?
     enum client_resize_mode client_resize_mode;
+
+    int pad1; /* unused; unicode_input_state */
+    enum xrdp_capture_code capture_code;
```

Comment:
The capture_code needs to go above the 'Private to xrdp below this line` comment. I'd suggest putting it just above the frame capture interval stuff.